docs/use-cases/manage-modules.md

```diff
@@ -1,277 +1,37 @@
 ---
-title: "Upload your Own Modules to the Viam Registry"
-linkTitle: "Upload your module"
-weight: 20
+title: "How to manage modules you created"
+linkTitle: "Manage modules"
 type: "docs"
-tags:
-  [
-    "server",
-    "rdk",
-    "extending viam",
-    "modular resources",
-    "components",
-    "services",
-  ]
-description: "Use the Viam CLI to upload a custom module to the Viam registry as a public module or as a private module that is shared only within your organization."
-no_list: true
-icon: true
-images: ["/registry/upload-module.svg"]
+weight: 26
+images: ["/registry/create-module.svg"]
+tags: ["modular resources", "components", "services", "registry"]
+description: "Update or delete your existing modules, or change their privacy settings."
 aliases:
-  - "/extend/modular-resources/upload/"
-  - "/modular-resources/upload/"
```

Comment:
I think we need to keep these still

Reply:
these two have moved to the other how-to